spec/event_spec.rb

```diff
@@ -130,6 +130,13 @@
       expect(events.count).to be 1
       expect(events.first).to be_instance_of Icalendar::Event
     end
+
+    it 'correctly parses a badly backslash file' do
+      input = File.read File.join(File.dirname(__FILE__), 'fixtures', 'backslash2.isc')
+
+      actual = described_class.parse(input)
+      expect(actual[0]).to be_a(described_class)
```

Comment:
It appears that this parsing is losing what is in the description.

If I add `expect(actual[0].description.to_s).to_not eq ""` to this test, it starts failing.